src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,37 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private K[] keys = (K[]) new Object[DEFAULT_CAPACITY];
+    private V[] values = (V[]) new Object[DEFAULT_CAPACITY];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
int value is 0 by default